tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -59,12 +59,14 @@ def test_cleanTas_space_nulls(tmpdir):
 
 
 def test_updateTASLookups(database, monkeypatch):
```

Comment:
You're right (and this is something @alburde1 pointed to as well). I'll see if I can make the test flow a bit more easily